lib/screens/player_screen.dart

```diff
@@ -174,17 +199,34 @@ class _PlayerScreenContent extends StatelessWidget {
                   return Column(
                     mainAxisAlignment: MainAxisAlignment.spaceAround,
                     children: [
-                      SizedBox(
-                          height: min(
-                              constraints.maxHeight -
-                                  controller.getTarget().height,
-                              constraints.maxWidth),
-                          width: constraints.maxWidth,
-                          child: const PlayerScreenAlbumImage()),
+                      SimpleGestureDetector(
```

Comment:
I'm planning to refactor this into using a PageView at some point, and I'm not sure if that would properly work from inside the `PlayerScreenAlbumImage` (because the next cover should slide in from off-screen). We also use that widget in some other places (now playing bar, queue panel), where we don't want it to be swipeable.  
So I think we should leave this here, and have code duplication instead?

Reply:
The PlayerScreenAlbumImage is only used here, the others just use AlbumImage.  I think any sort of track switching animation would need heavy refactoring of a lot of the components anyway and shouldn't weigh too heavily on decisions like this now, but code duplication is fine.

Reply:
I could've sworn that the widget was used in other places too, but you're right. I've moved the gesture detector there.  
We lose the "tap cover to go to start of song" functionality on the lyrics screen, but that was a minor and hidden feature anyway, and the new behavior is more consistent